Source/bmalloc/libpas/src/libpas/pas_platform.h

```diff
@@ -228,6 +228,11 @@
 #define PAS_HAVE_36BIT_ADDRESS 1
 #endif
 
+// PlayStation does not currently support the backtrace API. Android API versions < 33 don't, either. Windows does not either. Linux only with GLibc and not uCLibc/Musl.
+#if (PAS_OS(ANDROID) && __ANDROID_API__ >= 33) || (!PAS_PLATFORM(PLAYSTATION) && !PAS_OS(WINDOWS) && (PAS_OS(DARWIN) || (PAS_OS(LINUX) && defined(__GLIBC__) && !defined(__UCLIBC__))))
```

Comment:
I think there's no need to exclude PLAYSTATION and WINDOWS here, because they'll never match the other conditions:

```suggestion
#if (PAS_OS(ANDROID) && __ANDROID_API__ >= 33) || (PAS_OS(DARWIN) || (PAS_OS(LINUX) && defined(__GLIBC__) && !defined(__UCLIBC__)))
```

Reply:
Good point, I'll apply your suggestion before landing.